desktop/apps/auction_support/client/bid_form.coffee

```diff
@@ -12,10 +12,12 @@ module.exports = class BidForm extends ErrorHandlingForm
 
   timesPolledForBidPlacement: 0
   maxTimesPolledForBidPlacement: sd.MAX_POLLS_FOR_MAX_BIDS
-  errors:
+  errors: -> {
```

Comment:
It is surprising you need nested functions here. I would expect [this line](https://github.com/artsy/force/pull/1240/files#diff-1d073cf4803ffcbad07a7e898dfebad8R49) to be enough to do

````coffeescript
errors: -> {
  "Live Bidding has Started": "Foo #{@bar}"
}
````

Wrapping `errors` in a function should be enough to make it a method that runs lazily and has access to `this`.